_posts/2018-02-28-the-primary-owen-tracker.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: The Primary Owen Tracker
+date: 2018-02-28
+categories: projects
+description: The Primary Owen Tracker is a service for keeping track of the primary Owen of CSH. It allows people to suggest changes to the primary owen, and keep track of the current one. The rules for designating the primary owen is on Github!
+description: A Calendar Event Manager!
+image: /assets/img/default.jpg
+image-sm: /assets/img/default.jpg
+author: Skyler Clark
+author-image: https://avatars1.githubusercontent.com/u/4897492?s=460&v=4
+author-bio: A freshman computer science student!
+author-email: sjc1689@rit.edu
+author-social:
+  github: https://github.com/puffballinthedark
+  linkedin: https://www.linkedin.com/in/skyler-c-58962012b/
+---
+
+The Primary Owen Tracker is a webapp I made that sprung up around a joke regarding who the "primary Owen" is, or in simpler terms, the Owen who most recently defeated the other Owen in a challenge. 
+
+It first started out as a web page on whoistheprimaryowen.tk, which has since been taken down. I registered that domain in about ten minutes of free time, and pointed it to a single HTML document on my CSH server with one header that said in plaintext who the primary owen was. However, somewhere along the line, I started adding things to the website, and ended up with a pretty involved project for an in-joke. 
+
+The rules of the primary owen have evolved a bit since its first inception, but the gist of it is rougly the same. 
+
+you can become the primary owen if you either are already named Owen (or Rowan, who was voted in as an Owen), and beat the current primary Owen in any kind of challenge, or if you are not already named Owen,  by winning a challenge against all existing Owens.
+
+I made the website in flask and hosted it on Openshift, using a mysql database hosted by CSH. I learned how to make webapps, use flask as a backend, and work with web interfaces. I also learned how to use mysql datbases, git for source control, and Openshift to host projects.
+
+You can find it on github right [here](https://github.com/puffballinthedark/primaryOwentracker)!
```

Comment:
s/github/Github/

Reply:
Looks like it's technically [`GitHub`](https://en.wikipedia.org/wiki/GitHub), although as long as it's capitalized, that works too imo.